backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -469,7 +475,11 @@ public List<BehaviorGroup> getDefaultBehaviorGroups() {
     @Transactional
     @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN)
     public BehaviorGroup createDefaultBehaviorGroup(@NotNull @Valid BehaviorGroup behaviorGroup) {
-        return behaviorGroupRepository.createDefault(behaviorGroup);
+        BehaviorGroup createdBehaviorGroup = behaviorGroupRepository.createDefault(behaviorGroup);
+
+        // Sync new endpoint to evenType data model
+        endpointEventTypeRepository.refreshEndpointLinksToEventTypeFromBehaviorGroup(null, Set.of(createdBehaviorGroup.getId()));
```

Comment:
I'm not sure, because you can specify `actions` and `behavior` in `BehaviorGroup behaviorGroup` I think it could create all at once, since it will need to be sync with the new data model?

Reply:
Okay, let's leave it here. Worst case scenario, it does nothing.